ruby/cross_service_examples/item_tracker/src/report.rb

```diff
@@ -101,7 +101,7 @@ def post_report(recipient_email)
     work_items = @db_wrapper.get_work_items
     @logger.debug("Prepared the following items for a report:\n#{work_items}")
 
-    file_name = File.join(File.dirname(__FILE__), "templates", "report.html")
+    file_name = File.join(File.dirname(__FILE__), "templates", "report.erb")
```

Comment:
Sorry - should have elaborated more on this. This should be `report.html.erb` (convention for ERB to wrap underlying file name)